src/WattleScript.Interpreter/Execution/VM/Processor/Processor_InstructionLoop.cs

```diff
@@ -625,37 +625,37 @@ private void ExecExpTuple(Instruction i)
 
 		}
 
-		private void ExecSetPriv(Instruction i)
+		private void ExecSetFlags(Instruction i)
 		{
 			ref DynValue t = ref m_ValueStack.Peek(i.NumVal);
 			if (t.Type != DataType.Table)
-				throw new InternalErrorException("SETPRIV called on non-table object");
-			t.Table.PrivateKeys = new PrivateKeyInfo();
+				throw new InternalErrorException("SETFLAGS called on non-table object");
+			t.Table.Fields = new WattleFieldsInfo();
```

Comment:
This needs to be `??=` so that we can also add public members

Reply:
fixed in 6f5e0fe